app/components/event-card.js

```diff
@@ -20,19 +20,6 @@ export default Component.extend({
 
   didInsertElement() {
     this._super(...arguments);
-    const $innerSpan = this.$('.header > span');
-    const $header = this.$('.header');
-    while ($innerSpan.outerHeight() > $header.height()) {
-      $header.attr('data-content', $innerSpan.text());
-      $header.attr('data-variation', 'tiny');
-      $innerSpan.text((index, text) => {
-        return text.replace(/\W*\s(\S)*$/, '...');
-      });
-      $header.popup({
-        position: 'top center'
-      });
-      this.set('$header', $header);
-    }
   },
 
   willDestroyElement() {
```

Comment:
Didn't get you. To which component? I have moved this to the `smart-overflow` component instead of here.